### ISSUE TITLE
Misspelling on Education Page

### ISSUE THREAD

**Issue by Joel4763** [ProxymanApp/Proxyman#496]
On the Education page, https://proxyman.io/education, the word Licenses is spelled incorrectly, on the page it was spelled, Licenes, it should be spelled Licenses.

**Comment by NghiaTranUIT** [comment]
Good catch. Consider it's done 🙌

**Comment by Joel4763** [comment]
Awesome! You guys should ask GitHub about getting Proxyman on their GitHub Student Pack! I use the pack right now, and I also use your software as my goto proxy (Being that is it very simple and intuitive, but also feeling #Professional).

I was using mitmproxy, but I didn't really like the Web UI and the command line version was not easy, so I searched for an alt, and came across Proxyman!

As a Student, I use Proxyman for debugging my apps, API's, and my IOT devices!

In Conclusion, having Proxyman on GitHub Education Pack would benefit Students worldwide, with their proxying needs.

**Comment by NghiaTranUIT** [comment]
Yup, we're already requested to become a partner with Github Education Programm. This quarter is already finished and I have to wait for the next quarter with new batches 👍 

Btw, thank you for loving Proxyman ❤️ I'm going to re-write ProxymanCore v2 with Swift-NIO, which is better performance and less crashes too 😄

**Comment by Joel4763** [comment]
Awesome! Can't wait!!!!! 😊
